unclognearline/src/main/scala/streamcomponents/SetFlagsShape.scala

```diff
@@ -58,10 +59,16 @@ class SetFlagsShape extends GraphStage[FlowShape[UnclogStream, UnclogStream]]{
           if (deletable_count==elem.ParentProjects.length) {
             MediaStatusValue.DELETABLE
           } else if(deep_count>0) {
-            MediaStatusValue.SHOULD_BE_ARCHIVED
+            MediaStatusValue.SHOULD_BE_ARCHIVED_AND_DELETED
```

Comment:
didn't you way that SHOULD_BE_ARCHIVED_AND_DELETED means that completed_count==total_count and SHOULD_BE_ARCHIVED means that 1 < completed_count < total_count? can't see how that is applied here

Reply:
The correct logic is in the code.

If all projects are set to "Completed" the file should get the SHOULD_BE_ARCHIVED_AND_DELETED status. If more than one is set to "Completed" but not all are set to "Completed" then the file should get the SHOULD_BE_ARCHIVED status.

Reply:
how does that relate to `deep_count`?

Reply:
If none are set to deep archive the file should not be archived hence the code checking it.